shub/login.py

```diff
@@ -1,21 +1,56 @@
-import os, re, click
-from shub.utils import get_key_netrc, NETRC_FILE
+import re
+import click
+import os
+from shub import auth
+from scrapy.utils.conf import get_config
```

Comment:
yeah. =)
shub is for any Scrapinghub user -- not just the ones developing Scrapy projects.
I'd argue even in the current state it is already useful for people doing support. ;)